software/ever-gauzy.yml

```diff
@@ -0,0 +1,19 @@
+name: "Ever Gauzy"
+website_url: "https://gauzy.co"
+source_code_url: "https://github.com/ever-co/ever-gauzy"
+description: "Open Business Management Platform (ERP/CRM/HRM/ATS/PM)"
```

Comment:
> https://github.com/ever-co/ever-gauzy?tab=readme-ov-file#-what-is-it

```suggestion
description: "Open business management platform for collaborative, on-demand and sharing economies."
```

Reply:
@Ki-er think for most people ERP/CRM/HRM/ATS and PM mean more than "collaborative, on-demand and sharing economies". So because we have to keep the description very short (unlike how we do it in README), best if we keep it as "Open Business Management Platform (ERP/CRM/HRM/ATS/PM)".

Reply:
I have updated with (ERP/CRM/HRM/ATS/PM)